content/en/docs/tutorials/stateless-application/expose-external-ip-address.md

```diff
@@ -44,14 +44,11 @@ external IP address.
 
 1. Run a Hello World application in your cluster:
 
-{{< codenew file="service/load-balancer-example.yaml" >}}
+    {{< codenew file="service/load-balancer-example.yaml" >}}
```

Comment:
Please indent this by 3 spaces instead, i.e. align with the 'R' in 'Run' on line 45.

Be CAREFUL, do NOT indent 4 spaces, which is interpreted by the markdown parser to be something special.